dts/xtensa/intel/intel_cavs25.dtsi

```diff
@@ -251,5 +251,20 @@
 			status = "okay";
 		};
 
+		alh0:alh@24400 {
+			compatible = "intel,alh-dai";
+			reg = <0x00024400 0x00024600>;
+			label = "ALH_0";
+
+			status = "okay";
+		};
+
+		alh1:alh@244001 {
+			compatible = "intel,alh-dai";
+			reg = <0x00024400 0x00024600>;
```

Comment:
The first value here needs to match the value in line 262; otherwise you will get a build warning. Twister converts build warnings to errors but this PR somehow passes CI, so is there a sample or test application configuration that builds with this new driver?

Reply:
@MaureenHelm is there a "clean" way to create several instances of a driver with the same set of registers?

Reply:
Changed both of them to alh@24400, surprisingly it builds without warnings (previously there was a warning as you said), is that correct?

Reply:
Maybe twister only converts compiler warnings to errors, and not devicetree warnings too? What caught my attention in your previous version was the odd base address ending in 1. Also, is the second value here an address? It looks like an address, but it should be the size of the memory region.

Why are there multiple instances of the driver at the same address? How do you decide how many driver instances there should be? Is it a property of the hardware?

Reply:
> Maybe twister only converts compiler warnings to errors, and not devicetree warnings too? 

I remember I had to fix two issues in #41706 to stop `-Wa,--fatal-warnings` from being ignored. I see `--edtlib-Werror` in the same place but maybe it's being ignored too because of some other bug?

If it's not tested then it does not work and error handling is rarely ever tested. It took a number of coincidences to notice that `-Wa,--fatal-warnings` was being ignored.